snake/app.py

```diff
@@ -46,8 +47,10 @@ def run(self):
                 else:
                     filtered_events.append(event)
             self.currentScene.ProcessInput(filtered_events, pressed_keys)
-            self.currentScene.Update()
+            if time >= 60:
```

Comment:
This value is retrieved with the `clock.tick()` which returns the time in millisecond since the last time the function was called, so it is 60 milliseconds. You are right I will create a variable.